src/platform/monitoring/DowntimeNotification/config/scheduledDowntimeWindow.js

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment-timezone';
+
+export default {
+  downtimeStart: moment.tz('2020-01-07 00:01', 'America/New_York'),
+  downtimeEnd: moment.tz('2020-01-08 00:01', 'America/New_York'),
```

Comment:
Configure downtime window here. Only supporting one global downtime window for now. 

This is a **fallback** and it's never used when the api is available.

Reply:
So far this has made things difficult to test.  I'll do what I can to leave this pattern of a hardcoded downtime window while still allowing a test to change it.

Reply:
I personally like the idea of making this a static JSON file

Reply:
@ncksllvn is there an easy way to represent time zones with a static json? i guess we could break this up into parts with the date, time, and timezone strings.